src/binding.cpp

```diff
@@ -302,11 +302,17 @@ NAN_METHOD(render_file_sync) {
   NanReturnValue(NanNew<Boolean>(result == 0));
 }
 
+NAN_METHOD(libsass_version) {
+  NanScope();
+  NanReturnValue(NanNew<String>(libsass_version()));
```

Comment:
Yes, via the `libsass_version()` function. But now I see in the failing test Appveyor that it has always been set to `[NA]` on Windows and probably also on Unix (when compiled from the submodule) in the past.